backbone.stickit.js

```diff
@@ -609,18 +609,31 @@
           var currentVal = getAttr(model, options.observe, options);
           applyViewFn.call(this, options.update, $el, currentVal, model, options);
         };
+
+
         // We need to call this function after unstickit and after an update so we don't end up
         // with multiple listeners doing the same thing
         var removeCollectionListeners = function() {
           collection.off('add remove reset sort', refreshSelectOptions);
         };
+
         var removeAllListeners = function() {
           removeCollectionListeners();
           collection.off('stickit:selectRefresh');
+          collection.off('stickit:selectRefreshForView');
           model.off('stickit:selectRefresh');
         };
+
         // Remove previously set event listeners by triggering a custom event
-        collection.trigger('stickit:selectRefresh');
+        function removeCollectionListenersForView(cid) {
```

Comment:
Maybe name the argument `viewCid` just to be explicit *why* it's happening.  A comment here would be great too.